package.json

```
@@ -7,7 +7,8 @@
   "license": "GPL-3.0",
   "dependencies": {
     "@cypress/react": "^5.12.4",
-    "classnames": "^2.3.1",
+    "classname": "0.0.0",
+    "classnames": "^2.3.2",
```

```suggestion
    "classnames": "^2.3.2",
```
The first one is most likely redundant